src/main/java/menelaus/view/game/LevelSelectComponent.java

```diff
@@ -18,28 +18,53 @@
 import menelaus.model.LevelStars;
 import menelaus.model.SavedGames;
 
+/**
+ * The level view for the level select view.
+ * @author Obatola Seward-Evans
+ *
+ */
 public class LevelSelectComponent extends JPanel {
+	/** the level for this component. */
 	private Level level;
 	
+	/** the level stars for this level */
+	private LevelStars stars;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'stars' could be replaced by a local variable.](https://www.codacy.com/app/dmvouldjeff/menelaus/file/2446499320/issues/source?bid=3228196&fileBranchId=3246765#l31)